types/relay-runtime/lib/store/RelayModernQueryExecutor.d.ts

```diff
@@ -4,26 +4,41 @@ import {
     OperationTracker,
     OptimisticResponseConfig,
     PublishQueue,
+    ReactFlightPayloadDeserializer,
     SelectorStoreUpdater,
+    Store,
 } from './RelayStoreTypes';
 import { GraphQLResponse } from '../network/RelayNetworkTypes';
 import { Sink, RelayObservable } from '../network/RelayObservable';
 import { GetDataID } from './RelayResponseNormalizer';
 
+export type ActiveState = 'active' | 'inactive';
+
 export interface ExecuteConfig {
     getDataID: GetDataID;
+    treatMissingFieldsAsNull: boolean;
     operation: OperationDescriptor;
-    operationLoader: OperationLoader;
-    operationTracker?: OperationTracker;
-    optimisticConfig: OptimisticResponseConfig;
+    operationExecutions: Map<string, ActiveState>;
+    operationLoader: OperationLoader | null | undefined;
+    operationTracker?: OperationTracker | null | undefined;
+    optimisticConfig: OptimisticResponseConfig | null | undefined;
     publishQueue: PublishQueue;
-    scheduler?: TaskScheduler;
+    reactFlightPayloadDeserializer?: ReactFlightPayloadDeserializer | null;
```

Comment:
missed `| undefined`.  I know the `?` means `undefined` is allowed, but then we should be consistent with how `operationTrcker?` is written above

Reply:
I think you made `operationTracker` consistent in later commits